src/pages/new-claim/claim-evidence-instructions.tsx

```diff
@@ -0,0 +1,57 @@
+import Button from '../../components/button';
+import styles from './claim-evidence-instructions.module.scss';
+
+interface Props {
+  onNext: () => void;
+}
+
+export default function ClaimEvidenceInstructions(props: Props) {
+  return (
+    <div className={styles.container}>
+      <p>Your claim must include the following:</p>
+      <ul className={styles.terms}>
+        <li>
+          a) Show a direct connection between the usage of your subscribed data feed and the claimed malfunction,
+          including on-chain evidence of affected transactions
+        </li>
+        <li>b) Show permanent damages or losses caused by the malfunction</li>
+        <li>c) Calculation method of claimed relief amount</li>
+      </ul>
+      <p>
+        Please also note that your claim information linked within the claim cannot be altered after submission, and all
+        supporting evidence presented must be pinned on IPFS and publicly available at the time of Claim submission
+        until the Claim proceedings are concluded and the coverage policy reaches its end time.{' '}
+        <a href="https://docs.google.com" target="_blank" rel="noopener noreferrer">
```

Comment:
I've added https://linear.app/api3-dao/issue/DAO-165 to update these links when finalised and to add add the finalised graphic/image to this step